### DISCUSSION TITLE
Display error and keep loading in kindle browser

### DISCUSSION THREAD

**Discussion by kimkimtama** [Difegue/LANraragi#697] [Q&A]
I don't know if this is due to the capabilities of the Kindle WebKit browser. If yes, how can i work around it

**Comment by kimkimtama** [comment]
webkit version is 531.2+
![screenshot_2022_10_06T22_49_12+0800](https://user-images.githubusercontent.com/115158230/194345386-03bf6fc1-aabd-40d8-8c23-65fd52585f0e.png)

**Reply by Difegue** [reply]
👋 Does the Kindle browser even do javascript correctly? Sure it runs webkit but I've never tried it.  

You can try loading the oldstyle random archive URL `http://192.168.1.218:3000/random` to get directly into the reader -- maybe that'll work better?  
I'm afraid I don't really have a solution otherwise - I don't think Kindles accept external apps, so you can't even use the OPDS API. 🤔
